resources/_gen/assets/scss/scss/style.scss_f300667da4f5b5f84e1a9e0702b2fdde.content

```diff
@@ -542,7 +542,6 @@ pre {
   background: #FBFBFB; }
 
 .content * {
-  word-break: break-word;
   overflow-wrap: break-word; }
```

Comment:
This, unfortunately, didn't seem to update the preview for me (tried reloading cache and the like) maybe it didn't re-generate?